Cartesian_kernel/include/CGAL/Cartesian/Vector_3.h

```diff
@@ -65,7 +65,11 @@ class VectorC3
   { *this = R().construct_vector_3_object()(l); }
 
   VectorC3(const FT_ &x, const FT_ &y, const FT_ &z)
-    : base(CGAL::make_array(x, y, z)) {}
+    : base(Rep{x, y, z}) {}
+
+  template <class T1, class T2, class T3>
+  VectorC3(T1 &&x, T2 &&y, T3 &&z)
+    : base(Rep{std::forward<T1>(x), std::forward<T2>(y), std::forward<T3>(z)}) {}
```

Comment:
I didn't check if `: base{std::forward<T1>(x), std::forward<T2>(y), std::forward<T3>(z)} {}` would be ok or if we need to go through Rep explicitly.